pilot/proxy/envoy/testdata/lds-router-auth.json.golden

```diff
@@ -49,6 +49,86 @@
     ],
     "bind_to_port": true
    },
+   {
+    "address": "tcp://0.0.0.0:443",
+    "name": "http_0.0.0.0_443",
```

Comment:
Yeah, this should definitely go away. The moment there is an in-mesh HTTPS service (treated as TCP), envoy will either segfault or complain that there are multiple listeners on same port.